src/App.js

```diff
@@ -0,0 +1,57 @@
+import { $, routerPush } from "./utils.js";
+import weatherApi from "./service/WeatherApi.ts";
+import Component from "./core/Components.js";
+import WeatherInfo from "./components/WeatherInfo.js";
+import Header from "./components/Header.js";
+
+export default class App extends Component {
+  init() {
+    this.state = {
+      isPending: false,
+      isError: false,
+    };
+  }
+
+  template() {
+    return `
+    <div class='wrapper'>
+      <header></header>
+      <main>
+        <section id=outlet class=weather></section>
+      </main>
+    </div>`;
+  }
+
+  async mounted() {
+    const $header = $("header");
+    const $weather = $(".weather");
+    new Header($header);
+    const weatherInfo = new WeatherInfo($weather, {
+      clickEvent: () => routerPush("/weather/seoul"),
+    });
+    try {
+      const res = await weatherApi.getCurrentWeather({
+        q: "seoul",
+      });
+      weatherInfo.setState({
```

Comment:
P5;

가독성을 위해 줄바꿈을 넣어보면 좋을 것 같아요.
```suggestion
      });
      
      weatherInfo.setState({
```